src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    static class Node<K, V> {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private static class Node<K, V> {
```